projects/plugins/boost/app/assets/src/js/elements/Price.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+	import { getCurrencyObject } from '@automattic/format-currency';
+
+	export let value;
+	export let currency;
+	export let isOff;
+	export let hidePriceFraction;
+
+	const { symbol, integer, fraction } = getCurrencyObject( value, currency );
+	const showPriceFraction = ! hidePriceFraction || ! fraction.endsWith( '00' );
+</script>
+
+<p class="jb-price" class:jb-price--not-off-price={! isOff}>
```

Comment:
This is going to make me unpopular, but I'm looking at the `BenefitsInterstitial` and noting that it's already using the `PricingCard` element from Jetpack. That means we're already loading hundreds of kilobytes of React bloat.

Given that, we may as well use the React component where they're already built.

Given you've got this built, we can roll with this but I think in future we should lean more on premade React components.